src/main/java/com/snowflake/kafka/connector/internal/InternalUtils.java

```diff
@@ -105,14 +107,28 @@ static String timestampToDate(long time) {
     return date;
   }
 
+  /**
+   * Use this function if you want to create properties from user provided Snowflake kafka connector
+   * config.
+   *
+   * @param conf User provided kafka connector config
+   * @param sslEnabled is sslEnabled?
+   * @return Properties object which will be passed down to JDBC connection
+   */
+  static Properties createProperties(Map<String, String> conf, boolean sslEnabled) {
```

Comment:
can we add snowpipe something to this method name (createPropertiesForSnowpipe) or mention in the comments its for snowpipe?

Reply:
I didnt want make number of changes in test classes to reflect this change. I can add comment about using default ingestmethodconfig